src/test/java/com/easypost/AddressTest.java

```diff
@@ -0,0 +1,101 @@
+package com.easypost;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List;
+
+import com.easypost.exception.EasyPostException;
+import com.easypost.model.Address;
+import com.easypost.model.AddressCollection;
+import com.easypost.model.Batch;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class AddressTest {
+    
+    public static Address address;
+
+    @BeforeClass
+    public static void setup() throws EasyPostException{
+        EasyPost.apiKey = "EASYPOST_TEST_API_KEY";
+        address = Address.create(Fixture.basicAddress());
+    }
+
+    @Test
+    public void testCreate() throws EasyPostException {
+        Address address = Address.create(Fixture.basicAddress());
+
+        assertNotNull(address);
+        assertEquals("388 Townsend St", address.getStreet1());
+        assertTrue(address.getId().startsWith("adr_"));
+    }
+
+    @Test
+    public void testCreateVerifiyStrict() throws EasyPostException {
+        Map<String, Object> addressData = Fixture.basicAddress();
+        List<String> verificationList = new ArrayList<>();
+        verificationList.add("true");
+        addressData.put("verify_strict", verificationList);
+
+        Address address = Address.create(addressData);
+
+        assertNotNull(address);
+        assertEquals("388 TOWNSEND ST APT 20", address.getStreet1());
+        assertTrue(address.getId().startsWith("adr_"));
+    }
+
+    @Test
+    public void testRetrieve() throws EasyPostException{
+        Address retrievedAddress = Address.retrieve(address.getId());
+
+        assertNotNull(retrievedAddress);
+        assertEquals(address.getId(), retrievedAddress.getId());
+    }
+
+    @Test
+    public void testAll() throws EasyPostException{
+        Map<String, Object> params = new HashMap<>();
+        params.put("page_size", Fixture.pageSize());
+
+        AddressCollection addresses = Address.all(params);
+        List<Batch> addressesArray = addresses.getAddresses();
+        assertNotNull(addressesArray);
```

Comment:
Please split assertions and the logic to create/retrieve records with a line break. This makes it really easy for the eye to skim through these and get the info you need. This is true of all assertions everywhere. eg:

```
# Block of code creating stuff
1...
2...

# Block of code asserting stuff
1...
2...
```